src/components/carrossel-oficinas/styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+const VerMais = styled.div`
+  .styled-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #f5bc4a;
+    width: 146px;
+    height: 40px;
+    border: none;
+    margin-top: 1rem;
+  }
+  .escritaBotao {
```

Comment:
Outra coisa que lembrei agora, a consistência entre idiomas. Eu sugeriria este nome, mas é só uma ideia:

```suggestion
  .button-text {
```

Reply:
feito